### PR TITLE
fix(react-components): border imprint of split window widget and enabled dragging when minimized

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Bug 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/BND3D-3778

## Description :pencil:
Removed border imprint of split window widget, enabled dragging of split window in minimized state and add resizing in height (Y-axis) & width (X-axis)

## How has this been tested? :mag:

In Storybook

## Test instructions :information_source:

- `yarn && yarn build && yarn storybook`
- Select `SplitWindow` example
- When window is minimized, the window should be able to drag and in expanded state, the window is allowed to resize in X, Y & XY axis

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/Widgets/WindowWidget.tsx [hunk 1]
```diff
@@ -102,17 +105,13 @@ export const WindowWidget = ({
...
-        handle=".widget-header"
-        disabled={isMinimized}>
+      <Draggable onDrag={handleDrag} position={position} handle=".widget-header">
         <ResizableBox
           width={size.width}
           height={size.height}
           minConstraints={[WIDGET_WINDOW_MIN_WIDTH, WIDGET_INSIDE_WINDOW_MIN_HEIGHT]}
           maxConstraints={[parentContainerElement.clientWidth, parentContainerElement.clientHeight]}
-          resizeHandles={isMinimized ? [] : ['se']}
+          resizeHandles={isMinimized ? [] : ['se', 'ne', 'e', 's']}
```
Comment (haakonflatval-cognite) [inline react-components/src/components/Widgets/WindowWidget.tsx:114]: Is there a reason we want these in particular and not e.g. 'n' or `sw`? I also see that dragging the top-right corner upward, it only increases the height of the window downward. But not a critical bug

File: react-components/src/components/Widgets/WindowWidget.tsx [hunk 2]
```diff
@@ -54,6 +54,9 @@ export const WindowWidget = ({
   }
 
   const handleExpand = (): void => {
+    if (!isMinimized) {
+      setPosition({ x: 0, y: 0 });
```
Comment (haakonflatval-cognite) [inline react-components/src/components/Widgets/WindowWidget.tsx:58]: I'm not sure what this does :thinking: Does it put the window back to some specific position on screen every time it's expanded?

**Review by haakonflatval-cognite — APPROVED** [review]
A couple of comments but otherwise LGTM :+1:

File: react-components/src/components/Widgets/WindowWidget.tsx (hunk 2 above)
Comment (pramodcog) [reply react-components/src/components/Widgets/WindowWidget.tsx:58]: It places the window back to a specific position when minimized

File: react-components/src/components/Widgets/WindowWidget.tsx (hunk 1 above)
Comment (pramodcog) [reply react-components/src/components/Widgets/WindowWidget.tsx:114]: Increase in height downward while dragging top-right corner functionality is coming from `ResizeableBox` component, I could not find a way to fix it, will have a look into it latter.
And I haven't added `n, nw, sw, w` as these are increasing/decreasing height & width in one side that is down and right, it is not supporting top or left ward resize 🙅

**Pull request merged**